detection-rules/body_bec_mobile_solicitation.yml

```
@@ -10,7 +10,7 @@ source: |
                      "(email|send|shoot|give|provide|attach|confirm|) (me|your)"
   )
   and regex.contains(body.current_thread.text,
-                     "(cell(/s?phone)?|mobile.{0,10}(phone|number|#|no)|whatsapp)"
+                     "(cell(/s?phone)?|mobile.{0,10}|contact.{0,10}(phone|number|#|no)|whatsapp)"
```

```suggestion
                     "(cell(/s?phone)?|(mobile.{0,10}|contact.{0,10})(phone|number|#|no)|whatsapp)"
```
does this need to be this to maintain current parity? assuming this is for FNs

---

Don't think so, it's supposed to be something like `(cellphone OR mobile... OR contact...)` AND `(phone|number|#|no)|whatsapp`

---

Both regex's were looking a bit wonky. Updated both